src/index.ts

```diff
@@ -147,8 +147,17 @@ function generatePathMatches(paths: RMOAS.PathsObject, pathName: string, origin:
   return Object.keys(paths)
     .map(path => {
       const cleanedPath = normalizePath(path);
-      const matchStatement = match(cleanedPath, { decode: decodeURIComponent });
-      const matchResult = matchStatement(prunedPathName);
+
+      let matchResult: MatchResult;
+      try {
+        const matchStatement = match(cleanedPath, { decode: decodeURIComponent });
+        matchResult = matchStatement(prunedPathName) as MatchResult;
+      } catch (err) {
+        // If path matching fails for whatever reason (maybe they have a malformed path parameter)
+        // then we shouldn't also fail.
+        return;
```

Comment:
Is it ok to return nothing here when we're relying on the return output? Won't this generate an index with `undefined` in it?